packages/maskbook/src/plugins/Augur/Dashboard/index.tsx

```diff
@@ -0,0 +1,9 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { base } from '../base'
+
+const dashboard: Plugin.Dashboard.Definition = {
```

Comment:
it seems like this entry is not used. can you remove it?

Reply:
Nice catch, removed.